.github/workflows/health-issue-auto-closer.yml

```
@@ -0,0 +1,94 @@
+permissions:
+  issues: write
+  actions: read
+
+jobs:
+  auto-close-health-issues:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Check current health status
+        id: health_check
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:46: Double quote to prevent globbing and word splitting

(shellcheck)